packages/api/test/e2e/api.create.e2e.ts

```diff
@@ -32,8 +32,8 @@ describe('e2e api create', () => {
     });
 
     it('should create an Api instance with the timeout option', async () => {
-        const endPoint = 'wss://rimu.unfrastructure.io/public/ws';
-        api = await Api.create({provider: endPoint, timeout: 1000000000});
+        // const endPoint = 'wss://rimu.unfrastructure.io/public/ws';
```

Comment:
Should move this `endPoint` into a constant / config file and re-use appropriately.
Also make a note that we're testing against the local cennznet develop branch
I'll add a task to re-instate to re-configure the endpoint when a live network is up